app.py

```diff
@@ -28,15 +37,29 @@ class ReuploadDocbins(BaseModel):
 @app.post("/tokenize_record")
 def tokenize_record(request: Request) -> Tuple[int, str]:
     session_token = general.get_ctx_token()
-    value = util.tokenize_record(request.project_id, request.record_id)
+    value = tokenization_manager.manage_record_tokenization(
+        request.project_id, request.record_id
+    )
+    general.remove_and_refresh_session(session_token)
+    return value, ""
+
+
+@app.post("/tokenize_calculated_attribute")
+def tokenize_record(request: AttributeTokenizationRequest) -> Tuple[int, str]:
+    session_token = general.get_ctx_token()
+    value = task_manager.start_tokenization_task(
+        request.project_id, request.user_id, "ATTRIBUTE", request.attribute_name
```

Comment:
maybe a const or an enum for "ATTRIBUTE" & "PROJECT" as values for p3

Reply:
Inserted enums